File: code/java/Vectors.java
import java.util.Arrays;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.LinkedList;

public class Vectors {
	public static void main(String[] args) {
// BEGIN_CODE
        List<Integer> v =
            new ArrayList<>(Arrays.asList(5, 4, 3, 2, 1));
        List<Integer> l =
            new LinkedList<>(Arrays.asList(10, 9, 8, 7, 6));

        // sort the two
        Collections.sort(v);
        Collections.sort(l);

        // print them out
        for(Integer i : v) {
            System.out.print(i);
            System.out.print(',');
        }

        for(Integer i : l) {
            System.out.print(i);
            System.out.print(',');
        }
// END_CODE
	}
}

File: code/rust/vectors.rs
use std::collections::LinkedList;

fn main() {
// BEGIN_CODE
	let v = vec![5, 4, 3, 2, 1];
	let mut l = LinkedList::new();

    l.push_back(10);
    l.push_back(9);
    l.push_back(8);
    l.push_back(7);
    l.push_back(6);

    // sort the two
    v.sort();
    //l.sort();

    // print them out
    for i in v.iter() {
        print!("{},", i);
    }

    for i in l.iter() {
        print!("{},", i);
    }
// END_CODE
}

